pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/Converter.groovy

```diff
@@ -108,14 +112,28 @@ public class Converter {
      */
     public static ModelASTPipelineDef scriptToPipelineDef(String script) {
         CompilationUnit cu = new CompilationUnit(
-            CompilerConfiguration.DEFAULT,
+            makeCompilerConfiguration(),
             new CodeSource(new URL("file", "", DEFAULT_CODE_BASE), (Certificate[]) null),
             new GroovyClassLoader())
         cu.addSource(PIPELINE_SCRIPT_NAME, script)
 
         return compilationUnitToPipelineDef(cu)
     }
 
+    private static CompilerConfiguration makeCompilerConfiguration() {
+        CompilerConfiguration cc = new CompilerConfiguration();
+
+        ImportCustomizer ic = new ImportCustomizer();
+        ic.addStarImports(NonCPS.class.getPackage().getName());
+        ic.addStarImports("hudson.model","jenkins.model");
+        for (GroovyShellDecorator d : GroovyShellDecorator.all()) {
+            d.customizeImports(null, ic);
+        }
```

Comment:
Through the magic of `GroovyShellDecorator`s - https://github.com/jenkinsci/workflow-cps-global-lib-plugin/blob/master/src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryDecorator.java =)